newIDE/app/src/EventsBasedObjectEditor/EventBasedObjectChildrenEditor.js

```diff
@@ -231,7 +230,9 @@ export default class EventBasedObjectChildrenEditor extends React.Component<
                 // Nothing special to do.
                 onObjectCreated={() => {}}
                 onObjectSelected={this._onObjectSelected}
-                onRenameObject={(objectWithContext, newName, done) =>
+                renamedObjectWithContext={null}
+                onRenameStart={() => {}}
```

Comment:
This might be breaking the renaming of child objects in custom objects. Can you double check @ClementPasteau?
I think we need to add a state here.